distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -38,83 +65,107 @@ Function Update-openHAB {
-        Remove-Item ($userdata + '\etc\branding.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\branding-ssh.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\config.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\custom.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\custom.system.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\distribution.info') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\jre.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\org.apache.karaf*') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\org.ops4j.pax.url.mvn.cfg') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\profile.cfg') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\startup.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\system.properties') -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\etc\version.properties') -ErrorAction SilentlyContinue
-
-        Remove-Item ($userdata + '\cache') -Recurse -ErrorAction SilentlyContinue
-        Remove-Item ($userdata + '\tmp') -Recurse -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\all.policy') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\branding.properties') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\branding-ssh.properties') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\config.properties') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\custom.properties') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\distribution.info') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\jre.properties') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\org.ops4j.pax.url.mvn.cfg') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\profile.cfg') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\startup.properties') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\version.properties') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\etc\org.apache.karaf*') -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\cache') -Recurse -ErrorAction SilentlyContinue
+        Remove-Item ($OHDirectory + '\userdata\tmp') -Recurse -ErrorAction SilentlyContinue
 
         # We need to keep a backup in case the user modified that file
         Copy-Item ($userdata + '\etc\org.ops4j.pax.logging.cfg') -Destination ($userdata + '\etc\org.ops4j.pax.logging.cfg.bak') -Force
```

Comment:
These two line also need the `$OHDirectory +` base directory. Points to 'C:\etc\org.ops4j.pax.logging.cfg'

Reply:
This is backed up in the .zip file but I do now make a .bak copy in the correct location.